src/main/resources/alma/common/item.xml

```diff
@@ -32,14 +32,14 @@
     </entity>
     <!-- HOL if no ITM -->
     <entity name="" reset="true" sameEntity="true">
-      <!--TODO: Introduce filter that allows the entity to have separate objects. sameEntity="true" is the roadblock at the moment but we need it. 
+			<!--TODO: Introduce filter that allows the entity to have separate objects. sameEntity="true" is the roadblock at the moment but we need it. -->
       <if>
-          <none>
+          <none flushWith="record">
             <data source="ITM  .a">
               <buffer flushWith="HOL .8" />
             </data>
           </none>
-      </if> -->
+      </if>
```

Comment:
Only (DE-605)HT006855611has HOL without ITM, also it has multiple HOL's.
I assume that the if-conditional is now not working at all. This is the same problem I faced before. No HOL's are process at all instead of being filtered out. This is also due to the fact that ITM's are usually behind HOL.
Or do I miss the point here?